.github/workflows/verifyParserFiles.yml

```
@@ -0,0 +1,22 @@
+name: Check consistency of search parser files
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches-ignore: [staging, production]
+    paths:
+      - "src/libs/SearchParser/**"
+
+jobs:
+  verify:
+    if: github.actor != 'OSBotify' && github.actor != 'imgbot[bot]'
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: ./.github/actions/composite/setupNode
+
+      - name: Checking consistency
```

```suggestion
      - name: Verify parser files consistency
```

this name will make it more inline with the whole workflow name